package.json

```diff
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "scripts": {
+    "dev": "make dev",
```

Comment:
Any reason why we wouldn't directly stop using Makefile and set up whatever `make dev` does here?
 i.e.:
```suggestion
    "dev": "lerna run dev --stream --parallel",
```

Reply:
To be honest I didn't even look at what make dev was doing, just quickly added the script entry.

Reply:
i think this makes sense